README.md

```diff
@@ -114,6 +114,63 @@ An example cURL command to start the connector in distributed mode is:
     data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64", "channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
     ```
 
+#### Publishing messages with schema
+
+Ably Kafka connector supports messages with schema by converting schematic messages to JSON. Produced JSON is then passed as data to Ably message. JSON is produced by mapping schema field names to their values.
+For example if you use [Kafka Connect Avro Converter](https://www.confluent.io/hub/confluentinc/kafka-connect-avro-converter) and register an Avro schema to your [schema registry](https://docs.confluent.io/platform/current/schema-registry/connect.html), you can use the following. 
```

Comment:
Suggested change:

---
For example, if messages on the Kafka topic are serialized using Avro with schemas registered at `https://<your-schema-registry-host>`, then set the following configuration so that those messages are converted from Avro to JSON:

```
value.converter=io.confluent.connect.avro.AvroConverter
value.converter.schema.registry.url=https://<your-schema-registry-host>
```

Reply:
513e2e39a2373faf0be4792da8492b32bd68cd9d